launcher/nvdaLauncher.nsi

```diff
@@ -11,7 +11,6 @@ CRCCheck on
 
 ReserveFile "${NSISDIR}\Plugins\x86-unicode\system.dll"
 ReserveFile "${NSISDIR}\Plugins\x86-unicode\banner.dll"
-ReserveFile "..\miscDeps\launcher\nvda_logo.wav"
```

Comment:
can we also delete that file from `miscDeps` in a separate PR?

Reply:
Created nvaccess/nvda#35. Do we want to merge that and update the submodule on this branch?